docs/src/assets/readme.txt

```diff
@@ -0,0 +1,34 @@
+# Source
```

Comment:
if you use markdown then better name it readme.md (this is even better)

Reply:
Yeah, fixing it.

Reply:
I think it is enough to do a list of files we provide in assets. And for each file add information about the original source, the source from which we have taken the file and a link to the license we use (like Kaggle - they just give a link to the license)